### ISSUE TITLE
Incorrect URL to latest Geyser build

### ISSUE THREAD

**Issue by Holtster2000** [kejonaMC/GeyserUpdater#95]
Server log running GeyserUpdater 1.6.2:
```
[06:56:18] [Server thread/INFO]: [GeyserUpdater] A newer build of Geyser is available! Attempting to download the latest build now...
[06:56:18] [Craft Scheduler Thread - 2 - GeyserUpdater/ERROR]: [GeyserUpdater] Failed to download the newest build of Geyser
[06:56:18] [Craft Scheduler Thread - 2 - GeyserUpdater/WARN]: java.io.FileNotFoundException: https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/spigot/target/Geyser-Spigot.jar
[06:56:18] [Craft Scheduler Thread - 2 - GeyserUpdater/WARN]: 	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1996)
[06:56:18] [Craft Scheduler Thread - 2 - GeyserUpdater/WARN]: 	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1589)
[06:56:18] [Craft Scheduler Thread - 2 - GeyserUpdater/WARN]: 	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:224)
[06:56:18] [Craft Scheduler Thread - 2 - GeyserUpdater/WARN]: 	at GeyserUpdater-1.6.2.jar//com.projectg.geyserupdater.common.util.FileUtils.downloadFile(FileUtils.java:76)
[06:56:18] [Craft Scheduler Thread - 2 - GeyserUpdater/WARN]: 	at GeyserUpdater-1.6.2.jar//com.projectg.geyserupdater.spigot.util.GeyserSpigotDownloader.downloadGeyser(GeyserSpigotDownloader.java:87)
[06:56:18] [Craft Scheduler Thread - 2 - GeyserUpdater/WARN]: 	at GeyserUpdater-1.6.2.jar//com.projectg.geyserupdater.spigot.util.GeyserSpigotDownloader$1.run(GeyserSpigotDownloader.java:38)
[06:56:18] [Craft Scheduler Thread - 2 - GeyserUpdater/WARN]: 	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
[06:56:18] [Craft Scheduler Thread - 2 - GeyserUpdater/WARN]: 	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
[06:56:18] [Craft Scheduler Thread - 2 - GeyserUpdater/WARN]: 	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
[06:56:18] [Craft Scheduler Thread - 2 - GeyserUpdater/WARN]: 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[06:56:18] [Craft Scheduler Thread - 2 - GeyserUpdater/WARN]: 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[06:56:18] [Craft Scheduler Thread - 2 - GeyserUpdater/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:833)
```

File path used: `https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/spigot/target/Geyser-Spigot.jar`

Should be: `https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/spigot/build/libs/Geyser-Spigot.jar`

**Comment by Jens-Co** [comment]
Are you sure you got latest download from our Jenkins? 
https://ci.kejona.dev/job/GeyserUpdater/job/main/

has the got changed quite a while ago.

**Comment by Konicai** [comment]
Yeah, the download URL was fixed back in 5a4393a. Anyone experiencing this just has to redownload GeyserUpdater
